tests/unit/models/markup-section-test.js

```diff
@@ -149,3 +149,21 @@ test('#coalesceMarkers appends a single blank marker if all the markers were bla
   assert.equal(s.markers.length, 1, 'has 1 marker after coalescing');
   assert.ok(s.markers.head.isEmpty, 'remaining marker is empty');
 });
+
+test('.isBlank returns true if the text length is zero for two markers', (assert) => {
```

Comment:
Elsewhere in the tests we use `#` for instance method abbreviation and `.` (dot) for class method abbreviations. To stay in sync with those this should say `"#isBlank returns ..."`